gcn_classic_to_json/notices/GECAM_GND/__init__.py

```diff
@@ -0,0 +1,41 @@
+import numpy as np
+from scipy.stats import norm
+
+from ... import utils
+
+src_class_vals = ["GRB", "SFLARE", "KNOWN_SOURCE", "GENERIC_SOURCE"]
+
+containment_prob = norm().cdf(1) - norm.cdf(-1)
+
+
+def parse(bin):
+    bin[15]  # Unused. According to docs: '4 bytes for the future'
+    bin[20:39]  # Unused. According to docs: '76 bytes for the future'
+
+    soln_status_bits = np.flip(np.unpackbits(bin[18:19].view(dtype="u1")))
+    comments = None
+    if soln_status_bits[0] == 1:
+        comments = "This is a test notice."
+
+    return {
+        "mission": "GECAM",
+        "id": [bin[4]],
+        "messenger": "EM",
+        "mission_type": chr(bin[19] + 64),
+        # There is an error here; You are supposed to divide by 8640000 to get the correct value
+        # But it only seems to work if you divide by 864000000; I'm assuming it's some error in encoding the packets
+        "trigger_time": utils.datetime_to_iso8601(bin[5], bin[6] * 1e-2),
```

Comment:
Please check the C source code to determine if there is a mistake there.

Reply:
I believe this line in the C source code is the reason for this:
`lbuf[BURST_SOD] = (float)((event_sod*10000.0) + 0.00005);`
Usually it's multiplied by 100